src/App.js

```diff
@@ -1,26 +1,10 @@
-/* eslint-disable quotes */
-import logo from "./logo.svg";
-import "./App.css";
+/* eslint-disable quotes, jsx-quotes */
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App container '>
+      <Calculator />
```

Comment:
- [ ] It is required that all the components should be class-based components. Kindly change your App.js from functional to class-based component.

![project-rqt](https://user-images.githubusercontent.com/26597432/211640363-fb2746ca-5415-412d-9641-8af03e1d6c91.png)